atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -193,11 +195,13 @@ long grabLocksAndRun(SweepLocks locks) throws InterruptedException {
         } catch (RuntimeException e) {
             sweepMetrics.sweepError();
             if (checkAndRepairTableDrop()) {
-                log.error("The table being swept by the background sweeper was dropped, moving on...");
+                log.info("The table being swept by the background sweeper was dropped, moving on...");
```

Comment:
I would say this line is not an error, as sweep will recover properly from that situation, so I downgraded to info. Is my reasoning correct?